packages/flutter_test/lib/src/window.dart

```diff
@@ -272,15 +272,7 @@ class TestWindow implements Window {
   }
 
   @override
-  // use frameTimings. https://github.com/flutter/flutter/issues/38838
-  // ignore: deprecated_member_use
-  TimingsCallback get onReportTimings => _window.onReportTimings;
```

Comment:
Do these changes need to go through the API breaking change process?

https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

`onReportTimings` and `TimingsCallback` are very unlikely to have users, but we may want to email the announce list as part of this change.

Reply:
This is the `flutter_test/.../window.dart` instead of `lib/ui/window.dart`. Removing this doesn't remove the `onReportTimings` API (as it's inherited from `lib/ui/window.dart`) so this is not a breaking change.

Reply:
Ah, sorry, I did think this was in a `lib/` directory; nevermind re: the breaking change.

Reply:
The TestWindow is part of the flutter_test api. So this may break somebody who was using `tester.binding.window. onReportTimings` in their test. I don't think that is very common, but it may still be worth to send a quick note?